TACC/Core-CMS
# feat: tup 584 redesign news metadata

## Overview

Redesign news metadata.

### Status

- ~~fix [bug with short category, long author, and any date](https://github.com/TACC/Core-CMS/pull/773#issuecomment-1986601745)~~
    <sub>Leaving this as is. I do not want to refactor CSS grid to flex for this tiny bit of space. There is already a news CSS migration and a planned news CSS refactor coming. I can revisit then.</sub>
- [x] get designer review and approval

## Related

- [TUP-584](https://tacc-main.atlassian.net/browse/TUP-584)
- requires https://github.com/TACC/Core-Styles/pull/284
- required by https://github.com/TACC/tup-ui/pull/443

## Changes

See [all PRs merged to this branch](https://github.com/TACC/Core-CMS/pulls?q=is%3Apr+is%3Aclosed+base%3Afeat%2Ftup-584-news-metadata).

## Testing

1. Create article.
2. View article on list page.
3. View article page.
4. Test both pages responsiveness.
5. Confirm I get designer approval.

## UI

### Article Page

| Before | After |
| - | - |
| <img width="1155" alt="page before" src="https://github.com/TACC/Core-CMS/assets/62723358/ebe16116-7adb-4aeb-ba63-994c23d04ead"> | <img width="1155" alt="page after - desktop" src="https://github.com/TACC/Core-CMS/assets/62723358/3f1ee52b-418b-4fb3-a42e-d3158f23c62a"> |

| Mouse<br /><sup>e.g. typical Desktop</sup> | Touch<br /><sup>e.g. typical Mobile</sup> |
| - | - |
| <img width="1155" alt="page after - desktop" src="https://github.com/TACC/Core-CMS/assets/62723358/1dd8ce73-b3cc-455f-a7be-29b24bcdbf7b"> | <img width="1155" alt="page after - mobile" src="https://github.com/TACC/Core-CMS/assets/62723358/64231ffe-df0c-4d43-9728-cfa4be461255"> |

https://github.com/TACC/Core-CMS/assets/62723358/80417a9c-1103-4d5c-8eb4-923c37cd8227

### Article List Item

| Before | After |
| - | - |
| <img width="1194" alt="list before" src="https://github.com/TACC/Core-CMS/assets/62723358/c41f4289-2358-4194-881d-167a98667882"> | <img width="1194" alt="list after" src="https://github.com/TACC/Core-CMS/assets/62723358/10a84a18-5970-4b07-a87b-93d1255562a4"> |
